CONTRIBUTING.md

```diff
@@ -18,6 +18,8 @@ The service mesh landscape is powered by Jekyll. In order to add/update the land
 
 Entries should be listed in alphabetical order. Edit other yaml files as needed.
 
+- Aslo, to update the Service Mesh Timeline, add the new service mesh in the [non-functional.yml](https://github.com/layer5io/layer5/blob/master/_data/categories/non-functional.yml) file. Add `announce-date` key for the service mesh then add a `weight` key having value 1+`maximum value of weight`, to list the service mesh in the timeline correctly.
```

Comment:
Thank you. I don't know that the links in the list items above are still valid.

Reply:
Oops. They are not valid. Didn't check that earlier. Will update the links.